online.bib

```diff
@@ -0,0 +1,20 @@
+@online{bluej,
```

Comment:
What's the motivation of a separate file?

The disadvantage is that everyone has to include more files then. Furthermore, there is no -cleaned version of it.

Reply:
Not sure, I would be fine with adding it at the end for now.

Reply:
ok